docs/pages/enroll-resources/application-access/okta/app-and-group-sync.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Okta App and Group Sync
+description: Explains how to enable the Okta app and group sync integration, which imports Okta configurations into the Teleport RBAC system.
+---
+
+Okta has its own permissions system that doesn't directly map into Teleport. This
+includes permissions granted by Okta groups along with the assignment of users to individual
+applications within Okta. To model this within Teleport, an administrator would typically need
+to carefully craft a labeling system to attach to various Okta apps and groups and a set of
+roles to go along with them.
+
+With the synchronization of Okta apps and groups as Teleport Access List feature in the Okta
+integration, this work is performed for you. This guide explains how to set up
+Okta app and the sync.
+
+## How it works
+
+In Teleport, the Okta Access List synchronizer looks for any Okta group with
+members or any Okta application with individual assignments that matches filters
+that you can optionally supply during Okta integration enrollment. Access Lists
+will not be created for groups or applications without any assignments.
```

Comment:
We described when access lists will _not_ be created, but we never said when they will be created. I'd try to rephrase this section.